takwimu/templates/takwimu/_includes/search/index.html

```diff
@@ -63,72 +68,80 @@ <h1 class="display-4 text-center font-weight-bold">Please type something into th
+        position: relative;
+        padding-left: 1.5em;
+        cursor: pointer;
     }
 
     /* checkbox aspect */
     [type="checkbox"]:not(:checked) + label:before,
     [type="checkbox"]:checked + label:before {
-    content: '';
-    position: absolute;
-    left: 0; top: 0;
-    width: 1.25em; height: 1.25em;
-    border: 1px solid #9b9b9b;
-    background: #ffffff;
-    border-radius: 0;
-    /* box-shadow: inset 0 1px 3px rgba(0,0,0,.1); */
+        content: '';
+        position: absolute;
+        left: 0; top: 0;
+        width: 1.25em; height: 1.25em;
+        border: 1px solid #9b9b9b;
+        background: #ffffff;
+        border-radius: 0;
+        /* box-shadow: inset 0 1px 3px rgba(0,0,0,.1); */
     }
     /* checked mark aspect */
     [type="checkbox"]:not(:checked) + label:after,
     [type="checkbox"]:checked + label:after {
-    content: '✔';
-    position: absolute;
-    top: .1em; 
-    left: .1em;
-    font-size: 1.4em;
-    line-height: 0.8;
-    color: #000000;
-    transition: all .2s;
-    font-family: Helvetica, Arial, sans-serif;
+        content: '✔';
+        position: absolute;
+        top: .1em; 
+        left: .1em;
+        font-size: 1.4em;
+        line-height: 0.8;
+        color: #000000;
+        transition: all .1s;
+        font-family: Helvetica, Arial, sans-serif;
     }
     /* checked mark aspect changes */
     [type="checkbox"]:not(:checked) + label:after {
-    opacity: 0;
-    transform: scale(0);
+        opacity: 0;
+        transform: scale(0);
     }
     [type="checkbox"]:checked + label:after {
-    opacity: 1;
-    transform: scale(1);
-    }
-    /* disabled checkbox */
-    [type="checkbox"]:disabled:not(:checked) + label:before,
-    [type="checkbox"]:disabled:checked + label:before {
-    box-shadow: none;
-    border-color: #bbb;
-    background-color: #ddd;
-    }
-    [type="checkbox"]:disabled:checked + label:after {
-    color: #999;
-    }
-    [type="checkbox"]:disabled + label {
-    color: #aaa;
+        opacity: 1;
+        transform: scale(1);
     }
     /* accessibility */
     [type="checkbox"]:checked:focus + label:before,
     [type="checkbox"]:not(:checked):focus + label:before {
-    border: 1px dotted #000;
+        border: 1px dotted #000;
     }
 
     /* hover style just for information */
     label:hover:before {
-    border: 1px solid #4778d9!important;
+        border: 1px solid #4778d9!important;
     }
 
-</style>
\ No newline at end of file
+    /* Text wrappig */
+    .wrapword {
+        white-space: -moz-pre-wrap !important;  /* Mozilla, since 1999 */
+        white-space: -pre-wrap;      /* Opera 4-6 */
+        white-space: -o-pre-wrap;    /* Opera 7 */
+        white-space: pre-wrap;       /* css-3 */
+        word-wrap: break-word;       /* Internet Explorer 5.5+ */
+        white-space: -webkit-pre-wrap; /* Newer versions of Chrome/Safari*/
+        word-break: break-all;
+        white-space: normal;
+    }
+
+</style>
```

Comment:
This looks like should be globally available and therefore would be better suited in `takwimu.scss` if not already.

Reply:
@DavidLemayian 
Moved the css for checkbox in` takwimu.scss`